Doc/c-api/typeobj.rst

```diff
@@ -1587,7 +1587,7 @@ and :c:data:`PyType_Type` effectively act as defaults.)
    weak references to the type object itself.
 
    It is an error to set both the :c:macro:`Py_TPFLAGS_MANAGED_WEAKREF` bit and
-   :c:member:`~PyTypeObject.tp_weaklist`.
+   :c:member:`~PyTypeObject.tp_weaklistoffset`.
```

Comment:
Just to make it easy to review, the relevant check is in

https://github.com/python/cpython/blob/91d17305414923ae3f1cf98108ca42c50e60c8ea/Objects/typeobject.c#L7195-L7203

Reply:
Nice catch!
> I spotted a second error in the same section that seemed worth fixing at the same time.

I agree.
Still LGTM.